.github/workflows/workspace-integration-tests.yml

```diff
@@ -229,10 +237,22 @@ jobs:
                   SLACK_WEBHOOK: ${{ steps.secrets.outputs.WORKSPACE_SLACK_WEBHOOK }}
                   SLACK_COLOR: ${{ job.status }}
                   SLACK_MESSAGE: ${{ steps.test_summary.outputs.passed }}/${{ steps.test_summary.outputs.total }} tests passed
-            - name: Delete preview environment
-              if: github.event.inputs.skip_delete != 'true' && (success() || failure())
-              uses: ./.github/actions/delete-preview
-              with:
-                  name: ${{ needs.configuration.outputs.name }}
-                  sa_key: ${{ secrets.GCP_CREDENTIALS }}
-                  previewctl_hash: ${{needs.configuration.outputs.previewctl_hash }}
+
+    delete:
+      name: Delete preview environment
+      needs: [ configuration, infrastructure, check ]
+      if: github.event.inputs.skip_delete != 'true' && always()
```

Comment:
@vulkoingim the delete step did not run [here](https://github.com/gitpod-io/gitpod/actions/runs/4445311800), did you skip delete intentionally? Just checking :hugs:.

Reply:
Yeah, I ran it with skip. You can try to rerun it without

Reply:
Also, @vulkoingim , why `always()`?  [This](https://docs.github.com/en/actions/learn-github-actions/expressions#always) warns against it.

Reply:
Btw, great call splitting up the jobs, `failure()` [will still return true if any ancestor jobs fail](https://docs.github.com/en/actions/learn-github-actions/expressions#failure:~:text=If%20you%20have%20a%20chain%20of%20dependent%20jobs%2C%20failure()%20returns%20true%20if%20any%20ancestor%20job%20fails.), so our Slack Notify should "just work".

Reply:
> Also, @vulkoingim , why `always()`? [This](https://docs.github.com/en/actions/learn-github-actions/expressions#always) warns against it.

Because we want to always execute that job, regardless of the outcome of its dependencies